Cargo.toml

```diff
@@ -55,6 +55,9 @@ wasm-bindgen = "0.2.74"
 wasm-bindgen-futures = "0.4.24"
 web-sys = { version = "0.3.51", features = ["console"] }
 
+[patch.crates-io]
+lspower = {git = "https://github.com/influxdata/lspower.git", branch ="fix/close-client"}
```

Comment:
Do we want to wait for this to be released in lspower before merging? be careful not to delete the branch if you want to go ahead.

Reply:
Yes, I plan to wait for the patch to be release in lspower before merging, this PR is not a critical change to get merge since we have a workaround in place.